masterlist.yaml

```
@@ -1769,6 +1769,30 @@ plugins:
       - *alreadyInOrFixedByFCF
       - *alreadyInOrFixedByPRPv0_59
 
+  - name: '(MTM-NukaTownEastArchwayFix|MTM-GardenTerraceCeilingFixedByMonorail)\.es[lp]'
+    url: 
```

There's a trailing space here:

```suggestion
    url:
```